src/components/ConfigProvider/Theming/styleGenerator.ts

```diff
@@ -157,6 +160,32 @@ export function getStyle(themeOptions: ThemeOptions): IGetStyle {
     };
 }
 
+export function getFontStyle(fontOptions: FontOptions): IGetFontStyle {
+    let variables: Variables = {};
+
+    const font: OcFont = {
+        ...fontDefaults,
+        ...fontOptions.customFont,
+    };
+
+    // ================= Primary Font ==================
+    if (font.fontFamily) {
+        variables[`font-family`] = font.fontFamily;
```

Comment:
nit, the dynamic property names aren't really template strings.. can be standard single quotes